internal/server/http/handler/handler.go

```diff
@@ -0,0 +1,30 @@
+package handler
```

Comment:
i think we need work on this file

Reply:
I agree, as discussed, let's do that later, thanks :)